web/src/js/__tests__/ducks/ui/keyboardSpec.js

```diff
@@ -0,0 +1,158 @@
+jest.mock('../../../utils')
+
+import { Key } from '../../../utils'
+import { onKeyDown } from '../../../ducks/ui/keyboard'
+import reduceFlows from '../../../ducks/flows'
+import reduceUI from '../../../ducks/ui/index'
+import * as flowsActions from '../../../ducks/flows'
+import * as UIActions from '../../../ducks/ui/flow'
+import configureStore from 'redux-mock-store'
+import { createStore } from '../tutils'
+import { fetchApi } from '../../../utils'
+
+const mockStore = configureStore()
+console.debug = jest.fn()
+
+describe('onKeyDown', () => {
+    let flows = undefined
+    for( let i=1; i <= 12; i++ ) {
+        flows = reduceFlows(flows, {type: flowsActions.ADD, data: {id: i}, cmd: 'add'})
```

Comment:
We can use `{id: i, request: true, response: true}` for the flow to have better tests for left/right keys (https://github.com/mitmproxy/mitmproxy/blob/master/web/src/js/ducks/ui/keyboard.js#L64)